src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminResourceRoutes.scala

```diff
@@ -78,4 +90,33 @@ trait AdminResourceRoutes extends UserInfoDirectives with SecurityDirectives wit
       }
     }
 
+  private def requireActionsForAdminPutUserInPolicy(policyId: FullyQualifiedPolicyId, userInfo: UserInfo, samRequestContext: SamRequestContext)(addUserToPolicy: server.Route): server.Route =
+    requireOneOfAction(
+      FullyQualifiedResourceId(SamResourceTypes.resourceTypeAdminName, ResourceId(policyId.resource.resourceTypeName.value)),
+      Set(SamResourceActions.adminAddMember),
+      userInfo.userId,
+      samRequestContext
+    ) {
+      addUserToPolicy
+    }
+
+  private def requireActionsForAdminRemoveUserFromPolicy(policyId: FullyQualifiedPolicyId, userInfo: UserInfo, samRequestContext: SamRequestContext)(removeUserFromPolicy: server.Route): server.Route =
```

Comment:
This logic is borrowed from here: https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala#L389 which I _think_ is similar to what we want here?

Reply:
`requireOneOfAction` probably isn't needed here. Since we're only checking one action, you can use `requireAction` instead similar to https://github.com/broadinstitute/sam/blob/314eee8aae77032123df5264b6d484967c7e1843/src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala#L413

Reply:
Done!